lib/git_service/commit.rb

```diff
@@ -23,21 +23,63 @@ def rugged_diff(other_ref = parent_oids.first)
       other_commit.diff(rugged_commit)
     end
 
+    def formatted_author
+      "#{rugged_commit.author[:name]} <#{rugged_commit.author[:email]}>"
+    end
+
+    def formatted_author_date
+      rugged_commit.author[:time].to_time.strftime("%c %z")
+    end
+
+    def formatted_committer
```

Comment:
I mentioned in my commit message why I changed this here:

> [GitService::Commit] Fix Commit/CommitDate
> 
> In full_message, which is equivalent to the following `git-log` call:
>
>     $ git log -n1 --pretty=fuller --stat
>
> Has incorrectly defined `Commit` and `CommitDate` since it was added in
> commit 49b2146 of `ManageIQ/miq_bot`.
> 
> This addresses that, and handles `committer` in the same way that
> `author` has been formatted as well.

Reply:
Oh yeah that's a definite bug in the old one.